.github/workflows/actions_build.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - name: Build with Gradle (Shading only)
         run: |
           ./gradlew --no-daemon --stacktrace shadedJar shadedTestJar trimShadedJar \
-          ${{ startsWith(matrix.on, 'ubicloud') && '--max-workers=8' || '--max-workers=2' }} --parallel \
+          ${{ startsWith(matrix.on, 'ubicloud') && '--max-workers=12' || '--max-workers=2' }} --parallel \
```

Comment:
How about just keeping it as 8 even though there are 16 vCPU?
I think 8 workers are enough and we changed this machine due to the memory issue. 😉

Reply:
I thought we could reduce the cost a bit, but let's keep it at 8.